dataproc_jupyter_plugin/services/executorService.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict
+import requests
+from dataproc_jupyter_plugin.services.composerService import ENVIRONMENT_API
+from dataproc_jupyter_plugin.models.models import  DescribeJob
+import nbformat
+import subprocess
+from jinja2 import Environment,PackageLoader, select_autoescape
+import uuid
+from datetime import datetime, timedelta
+from dataproc_jupyter_plugin.utils.constants import CONTENT_TYPE, GCS
+import pendulum
+
+unique_id = str(uuid.uuid4().hex)
+job_id = ''
+job_name = ''
+TEMPLATES_FOLDER_PATH = "dagTemplates"
+ROOT_FOLDER = "dataproc_jupyter_plugin"
+def get_bucket(runtime_env, credentials,log):
+    if 'access_token' and 'project_id' and 'region_id' in credentials:
+            access_token = credentials['access_token']
+            project_id = credentials['project_id']
+            region_id = credentials['region_id']
+    api_endpoint = f"{ENVIRONMENT_API}/projects/{project_id}/locations/{region_id}/environments/{runtime_env}"
+
+    headers = {
+        'Content-Type': CONTENT_TYPE,
+        'Authorization': f'Bearer {access_token}'
+        }
+    try:
+        response = requests.get(api_endpoint,headers=headers)
+        if response.status_code == 200:
+                resp = response.json()
```

Comment:
Please run a python code formatter over the code so that formatting (including indentiation) is consistent.

The Jupyter [guidelines](https://jupyter-notebook.readthedocs.io/en/stable/contributing.html#code-styling) are to use https://github.com/psf/black

Reply:
Formatted backend files with black code formatter